### ISSUE TITLE
[Critical] Button: Button does not have a name

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#606]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87710 |
| **Checkpoint** | 4.1 Compatible › Name, Role, Value (4.1.2.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/11/26 at 9:43 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The button is missing an accessible name/label. As a result its functionality or purpose is not conveyed to screen reader users.
On Export Settings, select a few checkboxes, then Export.  In the resulting dialog, the close button is unlabeled.
Also occurs on New Export Question dialog (select Add) and Settings dialog (select Export).
See Import Export User Experience video at 8:50 and 14:15.

## Recommendation

Rule : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

How To Fix: 
Fix this issue by using any of the following techniques:
1. Use the innertext of the <button> or role="button" element

<button>Apply now!</button>

<button class="apply-btn"><span class="sr-text">Apply now!</span></button>

2. Use a non-empty aria-label attribute on the <button> or role="button" element.

<button class="apply-btn" aria-label="Apply now!"></button>

<span role="button" class="apply-btn" aria-label="Apply Now!"></span>

Reference: 
W3C ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/#aria-label

Background: 
Buttons must have discernible text that describes the button's function or action for screen reader and other assistive technology users.

## Screenshots

![02_button1](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87710/02_button1.png)
![02_button](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87710/02_button.png)
